src/api/solr-query.js

```diff
@@ -51,7 +51,9 @@ const fieldToQueryFilter = (field) => {
 	return null;
 };
 
-const buildQuery = (fields) => fields
+const buildQuery = (fields, mainQueryField) => fields
+	// Do not include main query field in filter field query param.
```

Comment:
There's some mixing of spaces and tabs here, but I think I'll resolve that in a different PR, as there's probably a lot to reconcile. I think the old project was using tabs, but React seems to go with spaces https://github.com/facebook/react/blob/master/.editorconfig.

Reply:
I'm happy to convert tabs to spaces for any file that's touched as part of this PR for now, if you'd like?